pymobiledevice3/services/crash_reports.py

```diff
@@ -22,8 +24,15 @@ def clear(self):
         """
         Clear all crash reports.
         """
+        undeleted_items = []
         for filename in self.ls('/'):
-            self.afc.rm(filename, force=True)
+            ret = self.afc.rm(filename, force=True)
+            if ret is not None:
+                undeleted_items.extend(ret)
+
+        for item in undeleted_items:
```

Comment:
Now need for the iteration
```python
if len(undeleted_items) > 0 and self.DEFAULT_FOLDER not in undeleted_items:
    raise # ....
```

Reply:
what? the function checking if there is file or dir except then the special case that is not deleted...

this suggestion only check if there is undeleted files **and** the special case not one of them (???)

Reply:
yeah, that's the same condition, just backwards